src/main/java/DeadLine.java

```diff
@@ -4,6 +4,7 @@ public class DeadLine extends Task
     public DeadLine (String taskName, String deadLineBy)
     {
```

Comment:
coding standard: Use K&R style brackets 
`public Deadline (String taskName, StringdeadLineBy) {`

Reply:
Thank you for pointing it out. This is a serious issue that i will resolve immediately.